src/main/java/springboot/Model/Instructor.java

```diff
@@ -1,13 +1,8 @@
-package Model;
+package springboot.Model;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Entity;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import java.util.ArrayList;
-import java.util.List;
+import javax.persistence.*;
 
-@Entity
+//@Entity
 public class Instructor extends Person {
 
     @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Does anyone know what this does, i.e. what the one to one relationship is with?

Reply:
One to one makes it so only one of this entity can map to one of another entity.